modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/patchedcell/v0.1.0.json

```diff
@@ -0,0 +1,197 @@
+              "description": "Seal resistance of patch",
+              "node": "bbpqsh:QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:pipetteResistance",
+              "name": "Pipette resistance",
+              "description": "Resistance of pipette used",
+              "node": "bbpqsh:QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:inputResistance",
+              "name": "Input resistance",
+              "description": "Input resistance of patched cell",
+              "node": "bbpqsh:QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:seriesResistance",
+              "name": "Series resistance",
+              "description": "Series resistance while patching cell",
+              "node": "bbpqsh:QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:liquidJunctionPotential",
+              "name": "Liquid junction potential",
+              "description": "Liquid junction potential of patch",
+              "node": "bbpqsh:QuantitativeValueShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:analogToDigitalConverter",
+              "name": "Analog-to-digital converter settings",
+              "description": "Settings of analog-to-digital converter",
+              "node": "this:AnalogToDigitalConverterShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:digitalToAnalogConverter",
+              "name": "Digital-to-analog converter settings",
+              "description": "Settings of digital-to-analog converter",
+              "node": "this:DigitalToAnalogConverterShape",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:labelingCompound",
+              "name": "Labeling compound",
+              "description": "The compound used in the pipette to label the cell",
+              "node": "bbptlotsh:BrainRegionOntologyTermShape",
+              "maxCount": 1
+            }
+          ]
+        }
+      ]},
+    {
+      "@id" : "this:AnalogToDigitalConverterShape",
+      "@type" : "sh:NodeShape",
+      "property" : [
+        {
+          "path" : "schema:unitCode",
+          "name" : "Unit",
+          "node":"bbptlotsh:UnitCodeOntologyTermShape",
+          "minCount" : 1,
+          "maxCount" : 1
+        },
+        {
+          "path" : "nsg:gain",
+          "name" : "Gain",
+          "datatype" : "xsd:float",
+          "minCount" : 1,
+          "maxCount" : 1
+        },
+        {
+          "path" : "schema:name",
+          "name" : "Name",
+          "description" : "Name of the Analog-to-digital converter",
+          "maxCount" : 1
+        }
+      ]
+    },
+    {
+      "@id" : "this:DigitalToAnalogConverterShape",
```

Comment:
The shape "this:DigitalToAnalogConverterShape" is very similar to "this:AnalogToDigitalConverterShape" except for "nsg:scale". I would suggest a refactoring so that on shape is created for unitCode, gain and name and reused in the two converter shapes.